client/pages/profile.jsx

```diff
@@ -0,0 +1,312 @@
+import { Button, Col, Input, Row, CardGroup, Card, CardText } from "reactstrap";
+import { Head } from "../components";
+import React, { useState, useEffect } from "react";
+import Link from "next/link";
+import "../public/styles/profile.scss";
+import Select from "react-select";
+import countryData from "../utils/countries";
+import "../public/styles/profileProjects.scss";
+import "../../api/public/schema/projectSchema";
+import { getUser, getModelsByID, getModels } from "../utils/apiWrapper";
+
+export default function Profile() {
+  const [user, setUser] = useState("");
+  const [createdProjects, setCreatedProjects] = useState([]);
+  const [followedProjects, setFollowedProjects] = useState([]);
+  const [projectParsed, setProjectParsed] = useState(false);
+  const [hasUser, setHasUser] = useState(false);
+  const DESCRIPTION_LENGTH = 150;
+  const [country, setCountry] = useState("");
+  const [projects, setProjects] = useState([]);
+
+  useEffect(() => {
+    const populateAllProjects = async () => {
+      const response = await getModels(null, true);
```

Comment:
Why do you load all of the projects here? Shouldn't we only use the users models down below?